static/app/utils/replays/replayReader.tsx

```diff
@@ -71,6 +73,9 @@ export default class ReplayReader {
     );
     replayRecord.startedAt = new Date(startTimestampMs);
     replayRecord.finishedAt = new Date(endTimestampMs);
+    replayRecord.duration = duration(
+      replayRecord.finishedAt.getTime() - replayRecord.startedAt.getTime()
+    );
```

Comment:
we re-calculated the start/finish times on the front-end, but not duration, derp.